packages/smooth_app/lib/generic_lib/widgets/smooth_card.dart

```diff
@@ -45,7 +45,10 @@ class SmoothCard extends StatelessWidget {
       elevation: elevation,
       shadowColor: const Color.fromARGB(25, 0, 0, 0),
       borderRadius: ROUNDED_BORDER_RADIUS,
-      color: color ?? Theme.of(context).colorScheme.surface,
+      color: color ??
+          (Theme.of(context).brightness == Brightness.light
```

Comment:
@M123-dev By curiosity, why did you remove the previous impl?

Reply:
@g123k with the color generation from a color swatch the color of the card got changed, to be grey and whiteish instead of black and white as they were before. 

For me this looks better (maybe just more familiar) but I tried to have as least impact on other widgets as possible as long as there is no custom color set.